packages/gravity-ui-web/src/sass/00-settings/_breakpoints.scss

```diff
@@ -0,0 +1,10 @@
+$grav-breakpoint-smallest: 32rem;
+$grav-breakpoint-largest: 75rem;
+
+/// Major layout breakpoints
+$grav-breakpoints: (
+  small: $grav-breakpoint-smallest,
+  medium: 44rem,
+  large: 56rem,
```

Comment:
What's the thinking about having variables for smallest and largest breakpoints but not for the medium/large?

Is there no situation where you'd need those? Just thinking out loud.

Reply:
Yeah, I just made those variables because I wanted a convenient way to reference those same values elsewhere.

I didn't have the same need for the other breakpoint widths, so I left them as is.